src/app/components/EmbedConsentBanner/ConsentBanner.styles.tsx

```diff
@@ -48,5 +48,11 @@ export default {
         border: `${pixelsToRem(1)}rem solid ${palette.POSTBOX}`,
         textDecoration: 'underline',
       },
+
+      // Applies focus indicator black outline.
+      // Overrides dotted Mozilla focus ring applied by Normalize global styles.
+      '&:focus-visible': {
+        outline: `3px solid black`,
```

Comment:
Could we convert the px value to rem please?

Reply:
Also not sure we usually write 'black'?

Reply:
Heya sorry I pushed this by accident - I updated [this in a later commit. ](https://github.com/bbc/simorgh/pull/10543/commits/eb5bc650b0b9f0b6f914c7614115fb17ce69811e)